src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -2647,7 +2647,7 @@ impl Coordinator {
             // Collect optimizer parameters.
             let optimizer_config = optimize::OptimizerConfig::from(self.catalog().system_config());
 
-            // Build an optimizer for this VIEW.
+            // (`optimize::view::Optimizer` has a special case for INSERTs.)
```

Comment:
I think the more germane comment is that it has a special case for *constants*. At least, I hope this to be true as part of merging the PR! :D

Reply:
Ok, changed it.